frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsPersistent.tsx

```diff
@@ -49,7 +49,8 @@ const RowInstanceActionsPersistent: FC<IRowInstanceActionsPersistentProps> = ({
     }
   };
 
-  return status === 'VmiReady' ? (
+  return status === 'VmiReady' ||
+    (status === 'ResourceQuotaExceeded' && running) ? (
```

Comment:
The instance is always running if the status is `ResourceQuotaExceeded`

Reply:
After the call to `mutateInstanceStatus` the value of `running` boolean flag changes.
I already tried without the second check, the result is that when the user taps the stop icon in the UI instance is correctly stopped, but the icon remains the same, instead of being changed in a start icon, so the user has no feedback about this operation

Reply:
Then there is something wrong with the operator, because the phase should become Off. That is probably related to the ignore if quota exceeded part, which indeed does not work.

Reply:
If the correct behavior of the operator is to set the phase to off I will fix this from tha backend